build.sbt

```diff
@@ -95,7 +95,7 @@ libraryDependencies ++= Seq(
     "com.gu" %% "content-api-client-aws" % "0.5",
     "com.gu" %% "content-api-client-default" % capiClientVersion,
     "com.gu" %% "editorial-permissions-client" % "2.0",
-    "com.gu" %% "fapi-client-play26" % "3.0.14",
+    "com.gu" %% "fapi-client-play26" % "3.0.16",
```

Comment:
Update to the correct version once https://github.com/guardian/facia-scala-client/pull/231 is officially released